akka-http-core/src/main/scala/akka/http/scaladsl/Http.scala

```diff
@@ -548,9 +548,12 @@ class HttpExt(private val config: Config)(implicit val system: ActorSystem) exte
   // every ActorSystem maintains its own connection pools
   private[http] val hostPoolCache = new ConcurrentHashMap[HostConnectionPoolSetup, Future[PoolGateway]]
 
+  // an ActorSystem-wide materializer must be used for gateways stored in cache
+  private[this] val hostPoolCacheFm = ActorMaterializer()(system)
```

Comment:
no need to configure this materializer? should it read settings from a separate config section, with fallback to the default?